addons/hr_skills/views/hr_views.xml

```diff
@@ -261,7 +261,10 @@
                     <group>
                         <group>
                             <field name="skill_type_id"/>
-                            <field name="skill_id" domain="[('skill_type_id', '=', skill_type_id)]"/>
+                            <field
+                                name="skill_id"
+                                domain="[('skill_type_id', '=', skill_type_id)]"
+                                options="{'no_create': True, 'no_create_edit':True}"/>
```

Comment:
`'no_create_edit': True` should be enough

Reply:
Indeed, fixed!